mpact/services.py

```diff
@@ -35,7 +36,7 @@
     WEBSOCKET_ROOM_NAME,
 )
 from telegram_bot.logger import logger
-from telegram_bot.utils import exception, increment_messages_count
+from telegram_bot.utils import exception, increment_messages_count, increment_message_count
```

Comment:
Can we drop `increment_messages_count` now?

Reply:
soon, i think. it's used in a few other places still.